python/ray/tune/tests/test_sample.py

```diff
@@ -59,10 +61,17 @@ def testTuneSampleAPI(self):
             self.assertGreaterEqual(out["randint"], -9)
             self.assertLess(out["randint"], 15)
 
+            self.assertGreaterEqual(out["lograndint"], 1)
+            self.assertLess(out["lograndint"], 10)
```

Comment:
Should we check if we actually have integers here (also for randint and qrandint)?

Reply:
Added a check for all of them